### PR TITLE
[MINI-4881] Add Interface for MAUID V2

### PR DESCRIPTION
**Pull request by munir-rakuten**
# Description
Added 2 Interfaces for `getMessagingUniqueId` and `getMauid` for MAUID V2.

## Links
MINI-4881

# Checklist
- [x] I have read the [contributing guidelines](../.github/CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](../.github/CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [x] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `./gradlew check` without errors

### CODE REVIEW

**Review by khairul-alam-licon — APPROVED** [review]
Looks good 👍

File: miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt [hunk 1]
```diff
@@ -243,6 +267,36 @@ open class MiniAppMessageBridge {
         bridgeExecutor.postError(callbackObj.id, "${ErrorBridgeMessage.ERR_UNIQUE_ID} ${e.message}")
     }
 
+    private fun onGetContactId(callbackObj: CallbackObj) = try {
```
Comment (corycaywood) [inline miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt:270]: See the updates on the iOS side - basically we wanted to rename this to `getMessagingUniqueId`.

Comment (munir-rakuten) [reply miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt]: addressed 👍

**Review by corycaywood — APPROVED** [review]
Looks good 👍

**Pull request merged**